### PR TITLE
Added better Font Combination to the UI and corrected typos

### PR DESCRIPTION
**Pull request by Deshmukh-Omkar**
Issue #63  
I have updated the font family and formatted all the pages accordingly.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
@Deshmukh-Omkar Please gitignore the .DS_Store files, unstage them and give the commits

**Comment by Deshmukh-Omkar** [comment]
What is this Greeting/greeting (pull_request)?

**Comment by Deshmukh-Omkar** [comment]
> @Deshmukh-Omkar Please gitignore the .DS_Store files, unstage them and give the commits

Ok, sure!

**Comment by smaranjitghose** [comment]
> What is this Greeting/greeting (pull_request)?



> What is this Greeting/greeting (pull_request)?

It's basically a GitHub Action that replies to you while we away for a long time

**Comment by anushbhatia** [comment]
Use uniform fonts all over it either Montserrat or Poppins

**Comment by Deshmukh-Omkar** [comment]
Sure, I'll incorporate all the requested changes.

**Comment by Deshmukh-Omkar** [comment]
@anushbhatia I have applied the Font Combination uniformly throughout the website.

File: assets/css/contact.css [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 /** bymeacoffee**/
 body{
-    font-family: 'Montserrat', sans-serif;
+    font-family: 'Lato', sans-serif;
```
Comment (anushbhatia) [inline assets/css/contact.css:6]: did you change this Montserrat or Poppins

**Comment by smaranjitghose** [comment]
Please fix the merge conflicts

File: assets/css/contact.css (hunk 1 above)
Comment (Deshmukh-Omkar) [reply assets/css/contact.css:6]: Hey @anushbhatia , I have implemented two fonts: Arvo (for the headings) and Lato (for the body) across all the pages as both of them together suit the theme of Doc2Pen. More over they contrast well with each other. 

Now, to maintain uniformity, I have changed fonts according to their role as Headings/Titles and Text. But I have accidentally changed the Buy me a coffee, as the original looks good. I'll unstage and correct it!

**Pull request closed without merging**